components/FAQ.tsx

```diff
@@ -0,0 +1,113 @@
+import React, { useState } from "react";
+
+const FAQ = () => {
```

Comment:
We generally do not split code into components if the component is not reusable

Reply:
Yes I agree, but I think we can fix this after exam phase right? :)

Reply:
Done.